pkgs/applications/office/morgen/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, stdenv, fetchurl, dpkg, autoPatchelfHook, makeWrapper, electron
+, alsa-lib, gtk3, libxshmfence, mesa, nss }:
+
+stdenv.mkDerivation rec {
+  pname = "morgen";
+  version = "210203";
+
+  src = fetchurl {
+    url = "https://dl.todesktop.com/${version}cqcj00tw1/linux/deb/x64";
+    sha256 = "sha256-ELSz66Bo492PVBrAxzykWN1q4vx0AAtuO8hi/0Hignk=";
```

Comment:
```suggestion
    sha256 = "sha256-rlXCZtIw0kkI2rxn41yW1dTAwwDgcqD/FET5fOP4DJg=";
```

Reply:
Good catch. I was going to upload it to the archive.org, but found that they offer the URL with the version, so I also changed it.